xml/ay_bigfile.xml

```diff
@@ -4680,6 +4680,27 @@ create_package_descr -x PATH_TO_EXTRA_PROV -d /usr/local/CDs/LATEST/suse</screen
 
    </sect2>
 
+   <sect2 xml:id="Software.Selections.recommend_packs">
+    <title>Installing Recommended Packages/Patterns</title>
+    <para>
+     While an installation all recommended packages/patterns will be installed.
```

Comment:
I would rephrase as: `By default all recommended packages/patterns will be installed.`